diff -p -up deskbar-applet-2.25.3/deskbar/handlers/mozilla.py.seamonkey deskbar-applet-2.25.3/deskbar/handlers/mozilla.py
--- deskbar-applet-2.25.3/deskbar/handlers/mozilla.py.seamonkey	2008-12-15 11:24:44.000000000 +0100
+++ deskbar-applet-2.25.3/deskbar/handlers/mozilla.py	2008-12-19 14:40:34.000000000 +0100
@@ -210,7 +210,7 @@ class MozillaBookmarksHandler(deskbar.in
     
     @staticmethod
     def has_requirements():
-        if is_preferred_browser("mozilla"):
+        if is_preferred_browser("seamonkey"):
             return True
         elif is_preferred_browser("firefox") or is_preferred_browser("iceweasel"):
             if MozillaBookmarksHandler.has_firefox_version():
@@ -253,7 +253,8 @@ class MozillaSearchHandler(deskbar.inter
                 "/usr/lib/mozilla-firefox/searchplugins",
                 "/usr/local/lib/mozilla-firefox/searchplugins",
                 "/usr/lib/iceweasel/searchplugins"] + \
-                glob.glob("/usr/lib*/firefox*/searchplugins")
+                glob.glob("/usr/lib*/firefox*/searchplugins") + \
+                glob.glob("/usr/lib*/seamonkey*/searchplugins")
         else:
             smart_dirs = [
                 get_mozilla_home_file("search"),
@@ -304,7 +305,7 @@ class MozillaSearchHandler(deskbar.inter
             # Correct firefox version or iceweasel
             MozillaSearchHandler.INSTRUCTIONS = _("You can customize which search engines are offered.")
             return True
-        elif is_preferred_browser("mozilla"):
+        elif is_preferred_browser("seamonkey"):
             # TODO - similar functionality for old-school mozilla (not firefox)
             return True
         else:
@@ -828,7 +829,7 @@ class MozillaHistoryHandler(deskbar.inte
         
     @staticmethod
     def has_requirements():
-        if is_preferred_browser("mozilla"):
+        if is_preferred_browser("seamonkey"):
             return True
         elif is_preferred_browser("firefox") or is_preferred_browser("iceweasel"):
             if MozillaBookmarksHandler.has_firefox_version():
